contracts/masset/liquidator/Liquidator.sol

```diff
@@ -17,119 +22,193 @@ import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
  * @author  mStable
  * @notice  The Liquidator allows rewards to be swapped for another token
  *          and returned to a calling contract
- * @dev     VERSION: 1.2
- *          DATE:    2020-12-16
+ * @dev     VERSION: 1.3
+ *          DATE:    2021-05-28
  */
-contract Liquidator is ILiquidator, Initializable, ImmutableModule {
+contract Liquidator is Initializable, ModuleKeysStorage, ImmutableModule {
```

Comment:
No interface applied here. Not a big deal since all are external calls

Reply:
The Liquidator interfaces are now different between the mainnet and Polygon chains so I updated the `contracts/masset/liquidator/ILiquidator.sol` interface to match the new mainnet Liquidator and add a new `contracts/polygon/IPLiquidator.sol` interface for Polygon.